modules/KalturaSupport/components/dualScreen/dualScreenControlBar.css

```diff
@@ -218,6 +218,9 @@
     top: 0;
     left: 0;
 }
+.mwPlayerContainer.mobile.mobileSkin .controlBar.dualScreen{
```

Comment:
@shalommeoded  this is nice and everything, yet you didn't delete the code that does the same thing

Reply:
@eitanavgil Sorry, a noob mistake...